src/app/layout.tsx

```diff
@@ -15,8 +15,8 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
+    <html lang="en" suppressHydrationWarning={true}>
```

Comment:
why are you using this there is a better alternative to it find it use that wherever is needed